sklearn/multioutput.py

```diff
@@ -0,0 +1,221 @@
+        check_consistent_length(X, y)
+        if y.ndim == 1:
+            y = y.reshape(-1, 1)
+        # Calculate the number of classifiers
+        self.n_outputs = y.shape[1]
+        self.estimators_ = []
+        for i in range(self.n_outputs):
+            estimator = clone(self.estimator)   # make a fresh clone
+            self.estimators_.append(estimator.fit(X, y[:, i]))
+
+        return self
+
+    def predict(self, X):
+        """Predict multi-class multiple target variable using a model
+         trained for each target variable.
+
+        Parameters
+        ----------
+        X : (sparse) array-like, shape = [n_samples, n_features]
+            Data.
+
+        Returns
+        -------
+        y : (sparse) array-like, shape = [n_samples, n_outputs]
+            Multi-output targets. An indicator matrix turns on multilabel
+            classification.
+            Predicted multi-class targets across multiple predictors.
+            Note:  entirely separate models are generated
+            for each predictor.
+        """
+        check_is_fitted(self, 'estimators_')
+
+        results = np.column_stack([estimator.predict(X) for estimator in
+                                   self.estimators_])
+        return results
+
+    def predict_proba(self, X):
+        """Probability estimates. This returns prediction probabilites
+        for each class for each label.
+
+        Parameters
+        ----------
+        X : array-like, shape = [n_samples, n_features]
+
+        Returns
+        -------
+        T : (sparse) array-like, shape = [n_samples, n_classes, n_outputs]
+            The class probabilities of the samples for each of the outputs
+        """
+        check_is_fitted(self, 'estimators_')
+
+        results = np.dstack([estimator.predict_proba(X) for estimator in
+                            self.estimators_])
+        return results
+
+    def score(self, X, y):
+        """"Returns the mean accuracy on the given test data and labels.
+
+        Parameters
+        ----------
+        X : array-like, shape = [n_samples, n_features]
+        y : array-like, shape = [n_samples, n_outputs]
+
+        Returns
+        -------
+        scores : array-like, shape = [n_outputs, 1]
+            Array of p_target floats of the mean accuracy
+            of each estimator_.predict wrt. y.
+        """
+        check_is_fitted(self, 'estimators_')
+        if y.ndim == 1:
+            y = y.reshape(-1, 1)
+        if y.shape[1] != self.n_outputs:
+            raise ValueError("The number of outputs of Y for fit {0} and"
+                             " score {1} should be same".
+                             format(self.n_outputs, y.shape[1]))
+        results = np.row_stack([estimator.score(X, y[:, i]) for i, estimator in
+                                enumerate(self.estimators_)])
+        return results
+
+    @property
+    def multilabel_(self):
```

Comment:
I don't think this is necessary. For starters estimators like LogisticRegression don't even have a `multilabel_` attribute